client_generator/main.go

```diff
@@ -310,7 +310,9 @@ func walk(p, url string, generator *clientGenerator, parent *clientType) {
 		fmt.Sprintf("/%s/types.ts", url),
 	)
 
-	ioutil.WriteFile(filename, []byte(ts), 0777)
+	if err := ioutil.WriteFile(filename, []byte(ts), 0777); err != nil {
```

Comment:
```suggestion
	if err := ioutil.WriteFile(filename, []byte(ts), os.ModePerm); err != nil {
```
漏れてました。
0777でしたらos.ModePermで！